frontend/13f-app/src/components/MainSection.css

```
@@ -0,0 +1,30 @@
+.main-container {
+    background: url('/mnt/c/Users/eharl/documents/repos/filing13F_searcher/frontend/13f-app/src/images/img-1.jpg') center center/cover no-repeat;
```

```suggestion
    background: url('../images/img-1.jpg') center center/cover no-repeat;
```
Please update this path, it will break in production.